qa/workunits/ceph-disk/ceph-disk.sh

```diff
@@ -1,13 +1,27 @@
 #!/usr/bin/env bash
-if [ -f $(dirname $0)/../ceph-helpers-root.sh ]; then
-    source $(dirname $0)/../ceph-helpers-root.sh
+BASEDIR=$(dirname $0)
+if [ -f $BASEDIR/../ceph-helpers-root.sh ]; then
+    source $BASEDIR/../ceph-helpers-root.sh
 else
-    echo "$(dirname $0)/../ceph-helpers-root.sh does not exist."
+    echo "$BASEDIR/../ceph-helpers-root.sh does not exist."
     exit 1
 fi
 
-install python-pytest || true
-install pytest || true
+PATH=$BASEDIR:$BASEDIR/..:$PATH
+
+: ${PYTHON:=python}
+$PYTHON --version
+
+# install pip if not present
+if ! type pip > /dev/null 2>&1 ; then
+    curl https://bootstrap.pypa.io/get-pip.py | $PYTHON
```

Comment:
woah, we are curling stuff from the internet? :( isn't it possible to install with normal packaging tools?

Reply:
Why is that :-( ? The machines are destroyed at the end of the test. . . 

The curl is not strictly needed, though, because - I found - pip is already present in all three OVH images we're presently using. I'm installing pytest via pip because I don't know how to install it via yum on RHEL (without entitlement subscription).

Reply:
There are several reasons why I don't think this is a good approach: 

* it isn't versioned, so results are not repeatable
* if results are not repeatable, then it opens up for unexpected behavior and possible breakage (pip has caused breakage before with incompatible flags)
* there is no way to verify that the curl is getting the file from a verified source, because there is no GPG signature to check against

Ephemeral nodes are great to minimize some problems, but it will not help with the above list of concerns. If we used the package manager to install `pip`, then none of the above would be a problem and it would still be able to install `pytest`.

If you say it isn't really needed, then why keep it?

Reply:
curl dropped!